### DISCUSSION TITLE
Cmd/Ctrl + V paste links without UI navigation

### DISCUSSION THREAD

**Discussion by jacksongoode** [WebCrateApp/webcrate#66] [Ideas] [answered]
From a Crate overview, it would be awesome to be able to simply paste in links with the keyboard.

**Comment by BetaHuhn** [comment, marked as answer]
@jacksongoode thanks for the suggestion!

This is now implemented on the [latest version of WebCrate](https://deta.space/discovery/@maxs1/webcrate) up on Space right now! Just paste any URL when WebCrate is open and after giving it access to your clipboard it will prompt you to save the pasted link. If you have a crate open it will add the link to that crate, otherwise you can select one.

**Reply by jacksongoode** [reply]
Fantastic news, thanks for letting me know!
